samples/highcharts/annotations/ellipse/demo.js

```diff
@@ -0,0 +1,152 @@
+                                            .pointToPixels(
+                                                target.points[0]
+                                            ),
+                                    attrs = target.getAttrsFromPoints();
+
+                                return {
+                                    x:
+                                        xy.x -
+                                        this.graphic.width / 2 -
+                                        attrs.rx *
+                                            Math.cos(
+                                                (attrs.angle * Math.PI) / 180
+                                            ),
+                                    y:
+                                        xy.y -
+                                        this.graphic.height / 2 -
+                                        attrs.rx *
+                                            Math.sin(
+                                                (attrs.angle * Math.PI) / 180
+                                            )
+                                };
+                            },
+                            events: {
+                                drag: function (e, target) {
+                                    var dx =
+                                            e.chartX -
+                                            (target.points[0].plotX +
+                                                target.chart.plotLeft),
+                                        dy =
+                                            e.chartY -
+                                            (target.points[0].plotY +
+                                                target.chart.plotTop),
+                                        newR = Math.max(
+                                            Math.sqrt(dx * dx + dy * dy),
+                                            5
+                                        ),
+                                        newAngle =
+                                            (-Math.atan(dx / dy) * 180) /
+                                                Math.PI -
+                                            90;
+
+                                    if (dy < 0) {
+                                        newAngle += 180;
+                                    }
+
+                                    target.setXRadius(newR);
+                                    target.setAngle(newAngle);
+                                    target.savePoints();
+
+                                    target.redraw(false);
+                                }
+                            }
+                        }
+                    ]
+                }
+            ]
+        }
+    ],
+    xAxis: {
+        minRange: 0.1
+    },
+    series: [
+        {
+            data: Array.from(
+                {
+                    length: 100
+                },
+                (_, x) => Math.floor(Math.cos(x / 10) * 20)
```

Comment:
Could you please explain the syntax here?

Reply:
first argument of the Array.from function is an object, that is passed, and the second one is index of this object, and i only want to use the second one. I use the `_` because first argument is irrelevent for me :)

Reply:
In this case, a good habit is using the `null` value

Reply:
I am not invoking the function here, but defining it, and the name of the variable can't be null.

Reply:
I get the concept here, `_` seams fine for me ;)